src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -199,6 +204,37 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
+    handleActiveAnnotation = (feedItems: FeedItems): void => {
+        const { emitAnnotatorActiveChangeEvent, file, match, onVersionChange } = this.props;
+        const { annotatorState: { activeAnnotationId } = {} } = this.state;
+        const { annotationId } = match.params;
+        const { fileVersionId } = match.params;
+        const currentFileVersionId = file.file_version.id;
+
+        // If annotatorState does not have an activeAnnotationId but the router path has an annotationId,
+        // then we are dealing with a hard load of a deeply linked annotation
+        if (activeAnnotationId || !annotationId) {
+            return;
+        }
+
+        if (currentFileVersionId === fileVersionId) {
+            // If the pathname matched file version is the current version of the file, attempt to set
+            // the active annotation in box-annotations to the specified annotationId
+            emitAnnotatorActiveChangeEvent(annotationId);
+        } else {
+            // If the pathname matched file version is an older version of the file, call onVersionChange
+            // to signal that we need to change the version
+            const selectedVersion = feedItems.find(item => item.type === 'file_version' && item.id === fileVersionId);
+
+            if (selectedVersion) {
+                onVersionChange(selectedVersion, {
+                    currentVersionId: currentFileVersionId,
+                    activeAnnotationId: annotationId,
```

Comment:
Alphabetize? Would also help to standardize on `active` versus `current` for both of these items.